app/components/PhotoUploadStatus/index.js

```diff
@@ -0,0 +1,126 @@
+//@flow
+import React, { Fragment, type ComponentType } from 'react';
+import Card from 'app/components/Card';
+import Icon from 'app/components/Icon';
+import Button from 'app/components/Button';
+import Flex from 'app/components/Layout/Flex';
+import { connect } from 'react-redux';
+import { Gallery } from 'app/actions/ActionTypes';
+import Tooltip from 'app/components/Tooltip';
+import {
+  selectGalleryPictureById,
+  type UploadStatus,
+  type GalleryPictureEntity,
+  initialUploadStatus
+} from 'app/reducers/galleryPictures';
+
+type StateProps = {
+  uploadStatus: UploadStatus,
+  lastImage: ?GalleryPictureEntity
+};
+
+type DispatchedProps = {
+  hideUploadStatus: () => void
+};
+
+type Props = StateProps & DispatchedProps;
+
+const style = {
+  zIndex: 2,
+  margin: 40,
+  position: 'fixed',
+  bottom: 0,
+  left: 0
+};
+const UploadStatusCard = ({
+  uploadStatus,
+  lastImage,
+  hideUploadStatus
+}: Props) => {
+  if (!uploadStatus.showStatus) return null;
+
+  const uploadDone =
+    uploadStatus.successCount + uploadStatus.failCount ===
+    uploadStatus.imageCount;
+
+  const hasFailedUploads = uploadStatus.failCount !== 0;
+
+  return (
+    <Card style={style}>
+      <Button onClick={hideUploadStatus}>
+        <Icon name="close" />
+      </Button>
+      {uploadDone ? (
+        <Fragment>
+          <h3>{uploadStatus.successCount} bilder ble lastet opp </h3>
+        </Fragment>
```

Comment:
What do you think about removing the popup a few seconds after the upload is done? Very "nice-to-have", but I think it could make sense.

Reply:
Guess we can do that! :smile: Out of scope of this PR tho.